### PR TITLE
add contribute-docs article

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Since we move our docs to our website, we need a docs to guide users how to contribute docs.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
cc @RainbowMango

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @RainbowMango

**Review by rgrupesh — COMMENTED** [review]
@Poor12 I think we should also add `setting up local environment section`.

**Comment by rgrupesh** [comment]
> > @Poor12 I think we should also add `setting up local environment section`.
> 
> Thanks. I'm a little confused about what you mean `setting up local environment section`

Like the prerequisite steps that contributor need to follow to clone the repository locally(on their machine) and contribute to it. In my view, if it detailed in the `contribute docs` it would help to avoid conflict in the future.

**Comment by Poor12** [comment]
Oh，I get it. Thanks for reminding.

File: docs/contributor/contribute-docs.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+---
+title: How to contribute docs
```
Comment (RainbowMango) [inline docs/contributor/contribute-docs.md:2]: @Tingtal Could you please take a look at this doc? you can get the preview at :
https://deploy-preview-145--karmada.netlify.app/docs/next/contributor/contribute-docs

File: docs/contributor/contribute-docs.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+If you add a document, you must add it to `sidebars.js` to make it display properly. If you're not sure where your docs are located, you can ask community members in the PR.
+
+### About Chinese docs
+
+There are two situations about the Chinese version of the document:
+* You want to translate our existing English docs to Chinese. In this case, you need to modify the corresponding file content from <https://github.com/karmada-io/website/tree/main/i18n/zh/docusaurus-plugin-content-docs/current>. 
+  The organization of this directory is exactly the same as the outer layer. `current.json` holds translations for the documentation directory. You can edit it if you want to translate the name of directory.
+* You want to contribute Chinese docs without English version. Any articles of any kind are welcomed. In this case, you can add articles and titles to the main directory first. Article content can be TBD first, like [this](../tutorials/karmada-search.md). 
+  Then Add the corresponding Chinese content to the Chinese directory.
```
Comment (Tingtal) [inline docs/contributor/contribute-docs.md:163]:
```suggestion
  Then add the corresponding Chinese content to the Chinese directory.
```

File: docs/contributor/contribute-docs.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+## Debugging docs
+
+Now you have already completed docs. After you start a PR to `karmada.io/website`, if you have passed CI, you can preview the effect of your document on the website.
```
Comment (Tingtal) [inline docs/contributor/contribute-docs.md:167]:
```suggestion
Now you have already completed docs. After you start a PR to `karmada.io/website`, if you have passed CI, you can get a preview of your document on the website.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/casestudies/adopters.md [hunk 4]
```diff
@@ -15,6 +15,6 @@ All organisations are sorted alphabetically.
 | Organization/Company Name | Website                 | Usage Scenario                                                                        | CaseStudy                                                                     |
 | --------------------------- | ------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
 | Hurricane Engines         | /                       | Use Karmada to build multi-cloud PaaS platform with multi-vendor and multi-region     | [Karmada in AIML INSTITUTE](ci123.md)                                         |
+| Trip.com                  | https://www.trip.com/   | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada          | TBD                                                                           |
```
Comment (Tingtal) [inline i18n/zh/docusaurus-plugin-content-docs/current/casestudies/adopters.md:18]:
```suggestion
| Trip.com                  | https://www.trip.com/   | Build a more resilient and elastic hybrid & multi-cloud platform using Karmada          | TBD                                                                           |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/casestudies/adopters.md [hunk 5]
```diff
@@ -15,6 +15,6 @@ All organisations are sorted alphabetically.
...
 | VIPKID                    | https://www.vipkid.com/ | Use Karmada to build multi-cluster platform with heterogeneous resources and clusters | [Building a PaaS Platform with Karmada to Run Containers --VIPKID](vipkid.md) |
 | Youzan                    | https://www.youzan.com/ | Manage both Stateless and Stateful on-line workloads across multiple clusters         | TBD                                                                           |
```
Comment (Tingtal) [inline i18n/zh/docusaurus-plugin-content-docs/current/casestudies/adopters.md:20]:
```suggestion
| Youzan                    | https://www.youzan.com/ | Manage both stateless and stateful online workloads across multiple clusters         | TBD                                                                           |
```

**Review by Tingtal — CHANGES_REQUESTED** [review]
Small touches on the language

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**